client-apis/metrics-api/src/main/java/org/wildfly/swarm/microprofile/metrics/cdi/AMetricRegistryFactory.java

```diff
@@ -1,21 +1,21 @@
 /*
- * Copyright 2017 Red Hat, Inc. and/or its affiliates
- * and other contributors as indicated by the @author tags.
  *
- *  Licensed under the Apache License, Version 2.0 (the "License");
- *  you may not use this file except in compliance with the License.
- *  You may obtain a copy of the License at
+ *   Copyright 2017 Red Hat, Inc, and individual contributors.
  *
- *     http://www.apache.org/licenses/LICENSE-2.0
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
  *
- *  Unless required by applicable law or agreed to in writing, software
- *  distributed under the License is distributed on an "AS IS" BASIS,
- *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- *  See the License for the specific language governing permissions and
- *  limitations under the License.
+ *   http://www.apache.org/licenses/LICENSE-2.0
  *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ * /
```

Comment:
Any idea why the copyright's are changing here?

Only expecting to see package renames

Reply:
rebasing trouble

Reply:
Is it fixable?

My concern is it's very hard to know what's really a change with this PR given that there are changes appearing that I wasn't expecting

Reply:
Well, if look at side by side it's equivalent contents: https://github.com/wildfly-swarm/wildfly-swarm/pull/708/files/8eb234ec52f3d214be70706337555c3d302ab4f4?diff=split#diff-bf4e418a4c75446f9b56e308113a0441